### DISCUSSION TITLE
Scaling vector graphics will be distorted,such as svg,why

### DISCUSSION THREAD

**Discussion by bai12123** [d2phap/ImageGlass#1553] [General]
Why scaling vector graphics, such as svg, will be distorted and blurred.This image in the preview list is larger and clearer than the scaled image.it's weird
![13  ：05  ：22](https://user-images.githubusercontent.com/99849852/232677776-357e3781-64a9-4d72-a365-b08e5bf9cd23.png)
@d2phap

**Comment by d2phap** [comment]
ImageGlass does not support scaling SVG yet but there is a plan for it in v9.0
https://github.com/d2phap/ImageGlass/issues/421
